### ISSUE TITLE
Exporting Library on a read-only NFS mount is a waste of time

### ISSUE THREAD

**Issue by stdedos** [xbmc/xbmc#18777]
### Describe the bug
Here is a clear and concise description of what the problem is:

If the NFS mount used is ro (which is nowhere indicated in Kodi), then exporting the library is a waste of time and erroneous log entries.

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

* It is somehow obvious that a NFS mount is ro
* Library export immediately skips ro mounts without trying, gives a warning "Skipped xyz because source is ro"

## Actual Behavior

Export Library will patiently try to export every single on of the entries

## Possible Fix

Try to touch a file. If it fails, skip the mount altogether.

### To Reproduce

1. 
2.
3.

### Debuglog

The debuglog can be found here:

### Screenshots 
Here are some links or screenshots to help explain the problem:

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

### Your Environment
Used Operating system:

 - [x] Android
 - [ ] iOS
 - [ ] tvOS
 - [ ] Linux
 - [ ] OSX
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name: 9
 - Kodi version: 18.9

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by KarellenX** [comment]
Sorry, I don't understand what bug you are reporting.

**Comment by stdedos** [comment]
I don't understand what you don't understand 😄
I did some touching-up on the description, maybe that helps?

**Comment by KarellenX** [comment]
Are you saying there is a bug because Kodi cannot write to a Read Only file system?

**Comment by stdedos** [comment]
I am saying:
* It wastes too much time trying to do so, repeatedly (when separate files are used).
* As a user, I am not informed that this is the case; I am only getting a generic "Error: Failed" notification ... 650 times, or more.

**Comment by basilgello** [comment]
Correct behavior here is to notify user with a dialog box once that the destination filesystem is read-only. It is not specific to NFS only: EXT4 volumes get remounted to read-only if `iget_checksum` verification fails and end-user sees many error toasts but has no clue what's oing on inside.

**Comment by DaVukovic** [comment]
> As a user, I am not informed that this is the case

As a user who created those exports or mounted them via the OS in use you should at least know either what and how is mounted or how you exported your NFS shares. 

But I have to admit that a better info in the log or a dialogbox would be somewhat nice. But this is not a bug, that's a feature request and should be submitted at the forums: https://forum.kodi.tv/forumdisplay.php?fid=9

I'll let other decide, but this is not a bug currently, but a missing information. And adding such an information is a feature request in my eyes. @KarellenX what do you think? Close this one and create a feature request or mark is as a valid bug?

**Comment by stdedos** [comment]
> As a user who created those exports or mounted them via the OS in use you should at least know either what and how is mounted or how you exported your NFS shares.

True, but I did this exactly 10 years ago, and I didn't remember it (somehow, "Sharing" and "NFS" options are separate ...). All I am saying is this is too assumptuous.

> But I have to admit that a better info in the log or a dialogbox would be somewhat nice. But this is not a bug, that's a feature request and should be submitted at the forums: [forum.kodi.tv/forumdisplay.php?fid=9](https://forum.kodi.tv/forumdisplay.php?fid=9)

I understand that; however, I am a little hesitant creating accounts everywhere for whatever unit whatever project picked for whatever purpose 😕 (especially forums that I'll definitely not do anything more than just make feature/bug reports).
